LinesWin.cs

```diff
@@ -24,12 +25,12 @@ class LinesWin : Window {
       RenderOptions.SetEdgeMode (image, EdgeMode.Aliased);
       Content = image;
       mDX = mBmp.Width; mDY = mBmp.Height;
-
+      FillPolygon ("C:/etc/leaf-fill.txt");

```

Comment:
Avoid hardcoded paths. Try to store this file as a resource in your application and use a manifest resource stream to load it from there.